src/terra-dev-site/guide/WhatIsTerra.a.guide.mdx

```diff
@@ -5,9 +5,11 @@ import MedicineIcon from '../icon/MedicineIcon';
 <div className="tui-illustration-grid">
 <div className="tui-illustration-grid-col">
 
-Terra UI offers a set of configurable [React components](https://engineering.cerner.com/terra-ui/#/components) designed to help build scalable and modular healthcare focused applications. The User Interface (UI) framework has been developed to work with Webpack and was created to solve real-world issues in projects we work on in the Healthcare IT space. Along with building out a UI component library, we've built out tooling to help with [testing](https://github.com/cerner/terra-toolkit), [development](https://github.com/cerner/terra-dev-site), and [managing translations](https://github.com/cerner/terra-aggregate-translations) when building React application.
+Build and integrate apps with Cerner.
```

Comment:
This feels like it's hanging out. Was there more or some context that was suppose to come before this?

Reply:
```suggestion

```

Reply:
It's intended to be the tag line of terra. Similar to Appkits "Construct and manage a graphical, event-driven user interface for your macOS app."
https://developer.apple.com/documentation/appkit

I'm going to make it a heading to make it stand out in the doc.